HappyAnding/HappyAnding/Views/ExploreShortcutViews/ExploreShortcutView/ExploreShortcutView.swift

```diff
@@ -75,6 +106,98 @@ struct ExploreShortcutView: View {
                 .presentationDragIndicator(.visible)
         }
     }
+    
+}
+
+// MARK: ViewBuilder
+extension ExploreShortcutView {
+    
+    @ViewBuilder
+    private func sectionView(with sectionType: SectionType) -> some View {
+        
+        lazy var shortcuts = sectionType.filteredShortcuts(from: shortcutsZipViewModel)
```

Comment:
let이 아니라 lazy var를 쓰신 이유가 궁금해요!
```suggestion
        let shortcuts = sectionType.filteredShortcuts(from: shortcutsZipViewModel)
```

Reply:
처음에 코드 작성할 때 데이터를 가져오는데 시간이 걸릴 거라고 생각했어요
그런데 저희가 앱 시작할 때 모든 데이터를 가져오기도 하고, 현재 shortcut이 아닌 vewmodel을 참조하고 있으니 let으로 선언해도 되겠네요! 
정말 감사해요! 수정하겠습니다 👍👍👍